gears/flywheel/breast-density-classifier.json

```diff
@@ -0,0 +1,46 @@
+{
+  "author": "support@flywheel.io",
+  "cite": "",
+  "command": "cd /opt/monai ; /opt/conda/bin/python app/breast_density_classifier_map.py -i /flywheel/v0/input/input-file/ -o /flywheel/v0/output/ -m models/traced_ts_model/traced_ts_model.pt",
+  "config": {
+    "save-json": {
+      "type": "boolean",
+      "default": false,
+      "description": "use flag to save json containing model results."
+    },
+    "debug": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enable debug mode."
+    }
+  },
+  "description": "Gear for breast density classification on mamography images",
+  "environment": {},
```

Comment:
expected that the environment key is empty ?

Reply:
@NPann sorry about that. this is fixed now.

Reply:
could you please merge after the build passes ?